docs/kpod-kill.1.md

```diff
@@ -0,0 +1,34 @@
+% kpod(1) kpod-kill- Kill one or more containers with a signal
+% Brent Baude
+# kpod-kill"1" "September 2017" "kpod"
+
+## NAME
+kpod kill - Kills one or more containers with a signal
+
+## SYNOPSIS
+**kpod kill [OPTIONS] CONTAINER [...]**
+
+## DESCRIPTION
+Kills one or more containers using a signal. The default signal is *KILL* but this can be overridden with the
+**-signal** command line switch. The containers can be identified by ID or name.
+
+## OPTIONS
+
+**--signal, s**
+
+Signal to send to the container. For more information on Linux signals, refer to *man signal(7)*.
+
+
+## EXAMPLE
+
+kpod kill mywebserver
+
+kpod kill 860a4b23
```

Comment:
Show an example of sending a different signal.

Reply:
there is one with TERM right below here... is that ok?

Reply:
If you have one that is fine.